postflight.d/sal-postflight

```diff
@@ -283,41 +283,40 @@ def get_puppet_report():
     return puppetreport
 
 
-def get_ohai_report():
+def get_ohai_report(insert_name):
     report = None
     ohai_path = '/opt/chef/bin/ohai'
+    ohai = dict()
+    new_ohai = dict()
     if os.path.exists(ohai_path):
-        command = [ohai_path, '-L', '/usr/local/sal/ohaiwarnings.log']
+        command = ['/opt/chef/bin/ohai']
         try:
             report = subprocess.check_output(command)
         except subprocess.CalledProcessError as error:
             munkicommon.display_debug2('Issue getting ohai report:')
             munkicommon.display_debug2(error.message)
-
-    ohai = {}
     if report:
         try:
             ohai = json.loads(report)
```

Comment:
changing this to `json.loads(report, object_pairs_hook=utils.dict_clean)` would probably solve your skip facts issue.